INTA governance (#124)
yabellini/SitioAcademico: content/blog/2024-01-07-INTA/index.md

---
title: Citizen science and governance models
author: Yanina Bellini Saibene
summary: " "
date: '2024-01-07'
categories:
  - English
  - Community
  - 100DaysToOffload
tags:
  - English
  - Community
  - 100DaysToOffload
---

> {{< figure src="featured.jpeg" alt="Field day at INTA. INTA's researcher showing soil trials and other related research activities and results">}}

Author: ```suggestion
{{< figure src="featured.jpeg" alt="Field day at INTA. INTA's researcher showing soil trials and other related research activities and results">}}

> Field day at INTA Anguil, La Pampa. INTA's researcher showing soil trials and other related research activities and results

```

> Continuing with the reflections on the governance models of organizations, communities of practice and research organizations, today I want to share with you the governance model of the National Institute of Agricultural Technology (INTA) where I worked as a researcher for 24 and a half years and how this related with Citizen Science.

Reviewer: "Continuing with" could link back to the previous post maybe?

> Continuing with the reflections on the governance models of organizations, communities of practice and research organizations, today I want to share with you the governance model of the National Institute of Agricultural Technology (INTA) where I worked as a researcher for 24 and a half years and how this related with Citizen Science.

Author: ```suggestion
[Continuing with](/blog/2024-01-05-cooperative/) the reflections on the governance models of organizations, communities of practice and research organizations, today I want to share with you the governance model of the National Institute of Agricultural Technology (INTA) where I worked as a researcher for 24 and a half years and how this related with Citizen Science.
```

## INTA 

INTA is an Argentine federal research and extension agency in charge of the generation, adaptation and diffusion of technologies, knowledge and learning procedures for the agriculture, forest and agro-industrial activities within an ecologically clean environment. It was created in 1956 and have 44 experimental agricultural stations and 240 extension units around the country.  It is the research institution with the largest territorial presence in our country. 

> In 1986, with the return to democracy in 1983, the president issued Decree 287, which modified the law that created INTA. The most important modifications were the _decentralization of the institute's operations_ and the **participation in its management of the different sectors involved in the generation and adoption of technology in each unit**.

Reviewer: "After the return to democracy in 1983, in 1986 the president issued Decree 287"

> In 1986, with the return to democracy in 1983, the president issued Decree 287, which modified the law that created INTA. The most important modifications were the _decentralization of the institute's operations_ and the **participation in its management of the different sectors involved in the generation and adoption of technology in each unit**.

Reviewer: Why `_decentralization_` but `**participation in its management**`? Why the different kinds of emphasis?

> In 1986, with the return to democracy in 1983, the president issued Decree 287, which modified the law that created INTA. The most important modifications were the _decentralization of the institute's operations_ and the **participation in its management of the different sectors involved in the generation and adoption of technology in each unit**.

Author: ```suggestion
After the return to democracy in 1983, in 1986 the president issued Decree 287, which modified the law that created INTA. The most important modifications were the _decentralization of the institute's operations_ and the **participation in its management of the different sectors involved in the generation and adoption of technology in each unit**.
```

> In 1986, with the return to democracy in 1983, the president issued Decree 287, which modified the law that created INTA. The most important modifications were the _decentralization of the institute's operations_ and the **participation in its management of the different sectors involved in the generation and adoption of technology in each unit**.

Author: ```suggestion
In 1986, with the return to democracy in 1983, the president issued Decree 287, which modified the law that created INTA. The most important modifications were the **decentralization of the institute's operations** and the **participation in its management of the different sectors involved in the generation and adoption of technology in each unit**.
```

The decree makes it clear that "In the regional operating units, the **balanced participation** of the different sectors of agricultural production, provincial governments, universities and the scientific-technical community **in their management will ensure the correspondence between the activities of the units and local problems.**" 

## Implementation

The INTA has a directive board composed of members of both the public and the private sector and with representation for different actors in the agricultural sector; they define the politics at national level. The national direction executes the planning of the board, assisted by the organization and administration technical areas.

> Each Regional and Research Center has a Regional board of director (I was the firs female member of the Natural Resource Research Center representing researcher that work at the experimental stations) that define the strategy ar regional level aligned with the national policy. Finally, the experimental stations and extension agencies have local advisory councils with the representation of local actors of the sector. All these governance body meet every month.  The National and Regional boards participate in the definition of the budget, the long- and medium-term strategic plan and the hiring of personnel. 

Reviewer: "first" not "firs"

Reviewer: "representing researchers" (plural)

Reviewer: It took me a while to read the first sentence - can you move the part in parentheses to a second sentence?

> Each Regional and Research Center has a Regional board of director (I was the firs female member of the Natural Resource Research Center representing researcher that work at the experimental stations) that define the strategy ar regional level aligned with the national policy. Finally, the experimental stations and extension agencies have local advisory councils with the representation of local actors of the sector. All these governance body meet every month.  The National and Regional boards participate in the definition of the budget, the long- and medium-term strategic plan and the hiring of personnel. 

Author: ```suggestion
Each Regional and Research Center has a Regional board of director that define the strategy at the regional level aligned with the national policy. I was the first female member of the Natural Resource Research Center, representing researchers who work at the experimental stations. Finally, the experimental stations and extension agencies have local advisory councils with the representation of local actors in the sector. All these governance bodies meet every month.  The National and Regional boards participate in the definition of the budget, the long- and medium-term strategic plan, and the hiring of personnel. 
```

For example, a regional board of director consists of:

1 member representing the national board of directors.
1 member representing the province goverment.
1 member representing the universities.
1 member representing the science-academic sector.
3 member representing the farmers.
1 member representing the local advisory councils.

## Science by and for the people

_Citizen Science_ definition say that it is research conducted with participation from the general public, or amateur/nonprofessional researchers or participants for science, social science and many other disciplines.[^1]

> Most of INTA's work involves citizen participation, for example, many field trials and technology tests are carried out on farmers' farms developed with with them, or with agricultural machinery manufacturers, many projects involve people recording field data and conducting surveys, but in my opinion **letting citizens be part of the leadership of science organizations** is the clearest and strongest signal that you really want them to be part of it and that their voice and opinion will be taken into account. 

Author: ```suggestion
Most of INTA's work involves citizen participation; for example, many field trials and technology tests are carried out on farmers' farms developed with them or with agricultural machinery manufacturers. Many projects involve people recording field data and conducting surveys, but in my opinion **letting citizens be part of the leadership of science organizations** is the clearest and strongest signal that you really want them to be part of it and that their voice and opinion will be taken into account. 
```

> It is not easy, there are frictions and conflicting interests among the sectors represented, but what better than to have them at the same table, dialoguing and agreeing on what is best for all.  What is better than knowing that your research and extension efforts are serving the people they will affect. What is better than a science at the service of the people. 

Reviewer: I would add a sentence here about how strange it is now to see science organizations that _aren't_ organized this way and that _don't_ put citizens at the center. Maybe the North has something to learn from the South... :-)

> It is not easy, there are frictions and conflicting interests among the sectors represented, but what better than to have them at the same table, dialoguing and agreeing on what is best for all.  What is better than knowing that your research and extension efforts are serving the people they will affect. What is better than a science at the service of the people. 

Author: ```suggestion
It is not easy; there are frictions and conflicting interests among the sectors represented, but what better than to have them at the same table, dialoguing and agreeing on what is best for all?  What is better than knowing that your research and extension efforts serve the people they will affect? What is better than a science at the service of the people? 

This is the way I learned to do science and the way I did science for most of my life.  It's very strange for me to see now science organizations around the world that _aren't_ organized this way and that _don't_ put citizens at the center.

Like the [previous article about cooperatives](/blog/2024-01-05-cooperative/), I write about INTA's governance and share it because it is a feasible model that has been working for 38 years in one of Argentina's largest science and technology institutions.  
```

---
title: Citizen science and governance models
author: Yanina Bellini Saibene
summary: " "
date: '2024-01-07'
categories:
  - English
  - Community
  - 100DaysToOffload
tags:
  - English
  - Community
  - 100DaysToOffload
---

{{< figure src="featured.jpeg" alt="Field day at INTA. INTA's researcher showing soil trials and other related research activities and results">}}

> Field day at INTA Anguil, La Pampa. INTA's researcher showing soil trials and other related research activities and results

> [Continuing with](/blog/2024-01-05-cooperative/) the reflections on the governance models of organizations, communities of practice and research organizations, today I want to share with you the governance model of the National Institute of Agricultural Technology (INTA) where I worked as a researcher for 24 and a half years and how this related with Citizen Science.

Author: ```suggestion
[Continuing with](/blog/2024-01-05-cooperative/) the reflections on the governance models of organizations, communities of practice and research organizations, today I want to share with you the governance model of the [National Institute of Agricultural Technology (INTA)](https://www.argentina.gob.ar/inta) where I worked as a researcher for 24 and a half years and how this related with Citizen Science.
```

## INTA 

INTA is an Argentine federal research and extension agency in charge of the generation, adaptation and diffusion of technologies, knowledge and learning procedures for the agriculture, forest and agro-industrial activities within an ecologically clean environment. It was created in 1956 and have 44 experimental agricultural stations and 240 extension units around the country.  It is the research institution with the largest territorial presence in our country. 

> After the return to democracy in 1983, in 1986 the president issued Decree 287, which modified the law that created INTA. The most important modifications were the _decentralization of the institute's operations_ and the **participation in its management of the different sectors involved in the generation and adoption of technology in each unit**.

Author: ```suggestion
After the return to democracy in 1983, in 1986 the president issued Decree 287, which modified the law that created INTA. The most important modifications were the **decentralization of the institute's operations** and the **participation in its management of the different sectors involved in the generation and adoption of technology in each unit**.
```

The decree makes it clear that "In the regional operating units, the **balanced participation** of the different sectors of agricultural production, provincial governments, universities and the scientific-technical community **in their management will ensure the correspondence between the activities of the units and local problems.**" 

## Implementation

The INTA has a directive board composed of members of both the public and the private sector and with representation for different actors in the agricultural sector; they define the politics at national level. The national direction executes the planning of the board, assisted by the organization and administration technical areas.

> Each Regional and Research Center has a Regional board of director that define the strategy at the regional level aligned with the national policy. I was the first female member of the Natural Resource Research Center, representing researchers who work at the experimental stations. Finally, the experimental stations and extension agencies have local advisory councils with the representation of local actors in the sector. All these governance bodies meet every month.  The National and Regional boards participate in the definition of the budget, the long- and medium-term strategic plan, and the hiring of personnel. 

Author: ```suggestion
Each Regional and Research Center has a Regional Board of Directors that defines the strategy at the regional level aligned with the national policy. I was the first female member of the Natural Resource Research Center, representing researchers who work at the experimental stations. 

Finally, the Experimental Stations and Extension Agencies have Local Advisory Councils with the representation of local actors in the sector. All these governance bodies meet every month. The National and Regional boards participate in the definition of the budget, the long- and medium-term strategic plan, and the hiring of personnel. 
```

For example, a regional board of director consists of:

> 1 member representing the national board of directors.

Author: ```suggestion
- One member representing the national board of directors.
```

> 1 member representing the province goverment.

Author: ```suggestion
- One member representing the province government.
```

> 1 member representing the universities.

Author: ```suggestion
- One member representing the universities.
```

> 1 member representing the science-academic sector.

Author: ```suggestion
- One member representing the science-academic sector (different from the universities).
```

> 3 member representing the farmers.

Author: ```suggestion
- Three members representing the farmers.
```

> 1 member representing the local advisory councils.

Author: ```suggestion
- One member representing the Local Advisory Councils.
```
